agent-inject/agent/agent.go

```diff
@@ -528,6 +535,17 @@ func (a *Agent) Patch() ([]byte, error) {
 			fmt.Sprintf("/spec/containers/%d/volumeMounts", i))...)
 	}
```

Comment:
Can this loop be removed now (lines 530-536)? It looks like it's duplicated below.

Reply:
Yes, sorry for that

Reply:
No worries!